packaging/installer/methods/manual.md

```diff
@@ -22,6 +22,9 @@ To install the latest git version of Netdata, please follow these 2 steps:
 
 ## Prepare your system
 
+Before you begin, make sure that your repo and the repo's submodules are clean from any previous builds and up to date.
+Otherwise, [perform a cleanup](https://github.com/netdata/netdata/edit/master/packaging/installer/methods/manual.md#perform-a-cleanup-in-your-netdata-repo)
```

Comment:
I see `edit` links everywhere, I think this was not intended. Please update the links.

Reply:
good eye, learn ingest process fixes them but you are correct